service-search-nls/src/main/java/fi/nls/oskari/search/channel/ELFGeoLocatorSearchChannel.java

```diff
@@ -440,4 +438,33 @@ public void calculateCommonFields(final SearchResultItem item) {
             item.setZoomScale(scale);
         }
     }
+
+    public List<String> doSearchAutocomplete(String searchString) {
+        //TODO Get from properties
+        String url = "http://54.195.254.125/asdielastic/placenames/placename/_search";
+        String requestJson = 	"{ \"query\": { \"match\": { \"name\": { \"query\": \""+searchString+"\", \"analyzer\": \"standard\" } } } };";
+        try {
+            //final String propertyPrefix = "search.channel." + getName() + ".service.";
+            log.info("Creating autocomplete search url with url:", url);
+            HttpURLConnection conn = IOHelper.getConnection(url);
+            String userpass = "asdiuser:3sto9AHT";
```

Comment:
Also you can pass user/pass to IOHelper.getConnection() and remove a few rows of code where you handle the header here.